### PR TITLE
Portuguese translation of README

### PR DESCRIPTION
**Pull request by KelvinCS**
### Description

That's the translation of README.md to Brazilian portuguese

### Affected components 

- Other (list it)
- README.md
- translations/README.md
- translations/README.portuguese.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,22 +42,27 @@ Read more at [hasura.io](https://hasura.io) and the [docs](https://docs.hasura.i
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
 **Table of Contents**
 
-- [Quickstart:](#quickstart)
+- [Hasura GraphQL Engine](#hasura-graphql-engine)
+  - [Features](#features)
+  - [Table of contents](#table-of-contents)
+  - [Quickstart:](#quickstart)
```
Comment (marionschleifer) [inline README.md]: We don't need this because it's content described before the TOC.

Comment (marionschleifer) [reply README.md]: Please remove this extra level of nesting.

File: translations/README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
-| File            | Language      |
-|-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) |
-| Event Triggers  | [:uk: English](../event-triggers.md)  ￨ [:fr: Français](event-triggers.french.md)  |
-| Remote Schemas  | [:uk: English](../remote-schemas.md)  ￨ [:fr: Français](remote-schemas.french.md)  |
-| Contributing    | [:uk: English](../CONTRIBUTING.md)    ￨ [:fr: Français](CONTRIBUTING.french.md)    |
-| Security        | [:uk: English](../SECURITY.md)        ￨ [:fr: Français](SECURITY.french.md)        |
-| Code of Conduct | [:uk: English](../code-of-conduct.md) ￨ [:fr: Français](code-of-conduct.french.md) |
+| File            | Language                                                                                                                                                            |
```
Comment (marionschleifer) [inline translations/README.md]: Why do we need that extra white space?

File: translations/README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (marionschleifer) [inline translations/README.md]: Why do we need these long lines? The table structure as it was before was quite nice 🙂

File: translations/README.portuguese.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+
+## Traduções
+
+Esse readme está disponível nas seguintes linguagens:
+
+- [English :uk:](.../README.md)
+- [Japanese :jp:](translations/README.japanese.md) (:pray: [@moksahero](https://github.com/moksahero))
+- [French :fr:](translations/README.french.md) (:pray: [@l0ck3](https://github.com/l0ck3))
+
+Traduções para outros arquivos podem ser encontradas [aqui](translations).
```
Comment (marionschleifer) [inline translations/README.portuguese.md]: Where does this link go?

File: translations/README.portuguese.md [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+* **Painel de Admin e Migrations**: Painle de Admin e esquema de migrations inspirado no Rails
+* **Postgres** ❤️: Suporta os tipos do Postgress (PostGIS/geo-location, etc.), transforma views em *graphs*, gatilhos e funções em mutations
+
+Leia mais em [hasura.io](https://hasura.io) e nas [docs](https://docs.hasura.io).
+
+## Table of contents
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Table of Contents**
+
+- [Hasura GraphQL Engine](#hasura-graphql-engine)
```
Comment (marionschleifer) [inline translations/README.portuguese.md]: Can you do the same changes here as above please?

**Comment by KelvinCS** [comment]
Thanks for the feedback! I will fix the problems above ASAP!

**Comment by KelvinCS** [comment]
I think It's done! Check it out

File: README.md [hunk 6]
```diff
@@ -42,22 +42,27 @@ Read more at [hasura.io](https://hasura.io) and the [docs](https://docs.hasura.i
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
 **Table of Contents**
 
-- [Quickstart:](#quickstart)
```
Comment (marionschleifer) [inline README.md]: I can still see some level of nesting in the table of contents. I think everything from line 45 to line 95 should be reverted to the original. Can you do that? The rest looks good 🙂

**Comment by marionschleifer** [comment]
@KelvinCS are you interested in implementing the above fixes? We have other PRs open for a Portuguese translation of the README.

**Pull request closed without merging**